packages/react-topology/src/components/TopologyControlBar/__snapshots__/TopologyControlBar.test.tsx.snap

```diff
@@ -665,21 +663,21 @@ exports[`TopologyControlBar should accept button options correctly 1`] = `
                 Object {
                   "current": <div
                     class="pf-c-toolbar__expandable-content"
-                    id="pf-topology-control-bar-1-expandable-content-1"
+                    id="undefined-expandable-content-1"
```

Comment:
Are these strings supposed to be evaluated as `undefined`?

Reply:
Nope. Updated!